ci-scripts/deploy-ccv2.sh

```diff
@@ -0,0 +1,149 @@
+echo "Verify server.ts has been updated for b2c dist"
+
+cat projects/storefrontapp/server.ts
+
+if grep -Fq "const distFolder = join(process.cwd(), 'dist/$B2C_STORE/browser');" projects/storefrontapp/server.ts
+then
+    echo "Dist folder has been updated"
+else
+    echo "Dist folder has NOT been updated"
+    exit 1
+fi
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Build SSR for b2c storefront"
+
+yarn build:ssr:ci
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Build CSR for b2c storefront"
+
+yarn build
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Copy server and browser files to js-storefront to adhere to the ccv2 dist structure for b2c storefront"
+
+cp -a dist/storefrontapp/. $CCV2_B2C_STOREFRONT_PATH/dist/$B2C_STORE/browser/
+cp -a dist/storefrontapp-server/. $CCV2_B2C_STOREFRONT_PATH/dist/$B2C_STORE/server/
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "update server.ts for b2b storefront"
+
+sed -i "s%dist/$B2C_STORE/browser%dist/$B2B_STORE/browser%gi" projects/storefrontapp/server.ts
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Verify server.ts has been updated for b2b dist"
+
+cat projects/storefrontapp/server.ts
+
+if grep -Fq "const distFolder = join(process.cwd(), 'dist/$B2B_STORE/browser');" projects/storefrontapp/server.ts
+then
+    echo "Dist folder has been updated"
+else
+    echo "Dist folder has NOT been updated"
+    exit 1
+fi
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Build SSR for b2b storefront"
+
+export SPA_ENV='b2b'
+yarn build:ssr:ci
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Build CSR for b2b storefront"
+
+yarn build
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Copy server and browser files to js-storefront to adhere to the ccv2 dist structure for b2b storefront"
+
+cp -a dist/storefrontapp/. $CCV2_B2B_STOREFRONT_PATH/dist/$B2B_STORE/browser/
+cp -a dist/storefrontapp-server/. $CCV2_B2B_STOREFRONT_PATH/dist/$B2B_STORE/server/
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Push to remote repository"
+
+cd $GHT_REPO
+
+git config --global user.email louis.pierrestiger@sap.com
```

Comment:
reason why it is louis email is because that's what is set in the global ccv2 script